libs/spec/Elf.3d

```diff
@@ -0,0 +1,460 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Ok for now but it seems lame that we have to copy this file from the everparse project to here rather than everparse supplying a set of common 3 files that projects can consume directly.  If there is an update then it requires multiple repos to take the update separately without coordination, which is error prone.

Reply:
Currently, the version of Elf.3d in the two repositories is the same. In the longer term, the version in our repository will be extended to include validation of other BPF-related sections in the ELF file that are not present in the generic ELF file. I asked the Everparse team, but there currently isn't a way to derive one spec from another, so forking is the only option.

Reply:
they're not quite the same... they differ in the code format pass (whitespace, capitalization, etc.) so it's not obvious whether they're the same or not.  I had to download both, diff them and manually go through all the diffs, which is a pain.   We should push back against the everparse team and file an issue in their repo.  We shouldn't block this PR on it, but we should file an issue I think.